kolena/_experimental/object_detection/workflow.py

```diff
@@ -52,6 +52,7 @@ def __post_init__(self):
 @dataclass(frozen=True)
 class Inference(BaseInference):
     bboxes: List[ScoredLabeledBoundingBox]
```

Comment:
I opted not to make `bboxes` an `Optional[List[ScoredLabeledBoundingBox]]` mostly because it makes for a really big diff elsewhere in all the downstream handling + requires thought for how it changes fields in the Studio UI. `List`s already have a decent empty state, so I don't think making a smaller change is too confusing.